js/app.js

```diff
@@ -0,0 +1,48 @@
+$(function(){
+
+  //listen for a click even on submit and do the function handleSubmit
+  $("#submitBtn").click(handleSubmit);
```

Comment:
@sofiaayala Does need to be called, the code is replicated in function `changeBackground`